extensions.sls

```diff
@@ -0,0 +1,16 @@
+# Install postgres extensions
+postgis:
+  pkg.installed:
+     - pkgs: [postgresql-{{ pillar['postgresql']['version'] }}-postgis-{{ pillar['postgresql']['postgis']['version'] }}]
```

Comment:
Mmh, this doesn't really work - having a generic extension state, but then installing specific packages?

Reply:
Yeah, kind of ugly. Howerver do we know that extensions are always:
'postgresql-{{ pillar['postgresql']['version'] }}-{{ extension }}-{{ pillar['postgresql']['postgis']['version'] }}' ?